catalog/resource_volumes_test.go

```diff
@@ -0,0 +1,65 @@
+package catalog
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	TestingName             = "testName"
```

Comment:
don't introduce variables for HTTP tests - they're harder to debug as HTTP fixtures.

instead, do inline those variables.